src/Providers/AbstractServiceProvider.php

```diff
@@ -106,6 +110,17 @@ protected function registerCacheProvider()
         });
     }
 
+    /**
+     * Register the bindings for the Driver.
+     */
+    protected function registerLogger()
+    {
+        $this->app->singleton('kolay-auth-logger', function ($app) {
+            $loggerStatus = (bool)$this->getConfig('logger_status');
+            return (new AuthLogger($loggerStatus));
+        });
+    }
+
```

Comment:
If i'm not wrong, you removed logs, so there is no need to keep these changes. Please remove everything related to log,

Reply:
i don't want to remove longer service

Reply:
Since it is a bug task and we should deploy it ASAP i'm going to resolve and approve this change. But you did a lot of un related changes and it is not a good practice to change where it is not needed. 
Next time i'm not going to accept your changes, even if it is bug.